include/ginkgo/core/matrix/csr.hpp

```diff
@@ -763,7 +763,10 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
 
     std::unique_ptr<Diagonal<ValueType>> extract_diagonal() const override;
 
-    std::unique_ptr<absolute_type> compute_absolute() const override;
+    std::unique_ptr<Csr<ValueType, IndexType>> create_submatrix(
+        const gko::span& row_span, const gko::span& column_span) const;
+
+    std::unique_ptr<absolute_type> compute_absolute() const;
```

Comment:
This seems to have gotten lost
```suggestion
    std::unique_ptr<absolute_type> compute_absolute() const override;
```